teps/0085-per-namespace-controller-configuration.md

```diff
@@ -79,6 +80,42 @@ As a contributor, I need to test my behavioral changes to ensure that they work
 - Operator can allow for configuration to be defined on per-namespace basis
 - User can specify and use a customized configuration for a given namespace
 
+## Proposal
+
+To enable the `tekton-pipelines-controller`'s feature flags configuration per namespace, we propose adding a new `configmap` with the same name `feature-flags` in each of the candidate namespaces, which contains the properties to override for all the `Pipelineruns` created in this specific namespace.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: feature-flags
+data:
+  disable-home-env-overwrite: "false"
+  scope-when-expressions-to-task: "true"
+```
+
+The candidate namespaces are included in a new environment variable `CONFIG_FEATURE_FLAGS_CUSTOMIZATION_NAMESPACES`.
```

Comment:
Please add one more sentence explaining this env. variable can be set in the pipeline-controller deployment resource. Thanks for adding example below 👍 

What happens when a namespace is listed in the deployment resource but does not exist in the system? Please add a little more explanation on such scenarios.

Reply:
just to add to @pritidesai comment it would be great to know what happen when this env variable is modified, describing what happen to existing resources and if only apply to newly created resources.